src/App.css

```diff
@@ -87,6 +87,20 @@ html,
   color: white;
 }
 
+.close-icon {
+  margin-left: 34rem;
+  cursor: pointer;
+}
+
+.button.is-cancel {
```

Comment:
I understand these .button.is-cancel and :hover styles are not needed anymore, right?

Reply:
Removed them!